packages/orn-locator/src/utils/acceptResponse.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { InvalidRequestError, NotFoundError, UnauthorizedError } from '@openstax/ts-utils/errors';
```

Comment:
ts-utils is also required elsewhere in orn-locator (in `src/resolvers/books.ts`), but it's not in orn-locator's package.json. Should we add it or leave it out?

Reply:
yea we should probably add it